copilot/fastapi/app/main.py

```
@@ -0,0 +1,153 @@
+        health_status["services"]["kubernetes"] = {
+            "status": "healthy"
+        }
+    except Exception as e:
+        health_status["services"]["kubernetes"] = {
+            "status": "unhealthy",
+            "error": str(e)
+        }
+        health_status["status"] = "degraded"
+    
+    # Check Argo CD connectivity
+    settings = get_settings()
```

Ruff (0.8.2)

Local variable `settings` is assigned to but never used

Remove assignment to unused variable `settings`

(F841)